README.md

```diff
@@ -306,6 +306,7 @@ Core rules are included in the `tslint` package.
 * `one-variable-per-declaration` disallows multiple variable definitions in the same statement.
   * `"ignore-for-loop"` allows multiple variable definitions in for loop statement.
 * `only-arrow-functions` disallows traditional `function () { ... }` declarations, preferring `() => { ... }` arrow lambdas.
+* `prefer-for-of` recommends a [TypeScript `for(... of ...)` loop](https://www.typescriptlang.org/docs/handbook/iterators-and-generators.html) over a standard `for` loop if the index is only used to access the array being looped over.
```

Comment:
Well so it is!  I was not aware, I thought it was just TypeScript syntactic sugar.